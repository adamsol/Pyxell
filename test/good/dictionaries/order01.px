
print {'a': 3, 'c': 9, 'b': 2} + {'a': 5, 'x': 0} + {'b': 1}
