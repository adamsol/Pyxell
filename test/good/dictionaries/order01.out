{'a': 5, 'c': 9, 'b': 1, 'x': 0}
